mne_nirs/statistics/tests/test_statsmodels.py

```diff
@@ -11,7 +13,12 @@
 from ...utils._io import glm_to_tidy
 
 
-def test_statsmodel_to_df():
+@pytest.mark.parametrize('func', ('mixedlm', 'ols'))
```

Comment:
Would it be possible to also test rlm here too (I'm away from PC so cant check)?

```suggestion
@pytest.mark.parametrize('func', ('mixedlm', 'ols', 'rlm'))
```